### PR TITLE
Use dotnet-format instead of broken dotnet format

### PR DESCRIPTION
**Pull request by nwithan8**
# Description

- Swap linting utility for one that isn't broken 🙄 

# Testing

- N/A

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,7 +27,7 @@ coverage:
 
 ## format - Formats the project
 format:
-	dotnet format
+	dotnet dotnet-format --no-restore
```
Comment (Justintime50) [inline Makefile:30]: Question: Is it not just `dotnet-format` instead of `dotnet dotnet-format`? Sad if it is... redundant

File: Makefile [hunk 2]
```diff
@@ -36,13 +36,20 @@ format:
...
 	scripts\install_cert.bat ${cert} ${pass}
 
-## install-scanner - Install SecurityCodeScan to your system
-install-scanner:
-	dotnet tool install --local security-scan --version 5.6.3
+## install-tools - Install required dotnet tools
+install-tools:
+	dotnet new tool-manifest || true
+	dotnet tool install --local security-scan --version 5.6.3 || true
+	dotnet tool install --local dotnet-format || true
```
Comment (Justintime50) [inline Makefile:43]: Prefer `exit 0` for fallbacks as it more clearly defines our expected outcome.

File: Makefile (hunk 1 above)
Comment (nwithan8) [reply Makefile:30]: `dotnet` brings up the dotnet CLI utility. From there, we tell it to use the `dotnet-format` tool.
You can also summon `dotnet-format` directly if it was installed globally. Our tools are installed locally, so no.

File: Makefile (hunk 2 above)
Comment (nwithan8) [reply Makefile]: Resolved

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**